--- a/htdocs/luci-static/resources/view/irqbalance.js
+++ b/htdocs/luci-static/resources/view/irqbalance.js
@@ -66,7 +66,7 @@
 
 	cells.push(E('div', { 'class': 'td right', 'data-title': _('Total') },
 		row.total ? '%d'.format(row.total) : '-'));
-	cells.push(E('div', { 'class': 'td', 'data-title': headings.last }, row.tail || '-'));
+	cells.push(E('div', { 'class': 'td', 'data-title': headings.last }, row.tail ? _(row.tail) : '-'));
 
 	const tr = E('div', {
 		'class': 'tr cbi-rowstyle-%d'.format(index % 2 ? 2 : 1),
